CliWrap/Cli.cs

```diff
@@ -94,13 +96,18 @@ public ExecutionOutput Execute(ExecutionInput input, CancellationToken cancellat
                 // Wire events
                 process.OutputDataReceived += (sender, args) =>
                 {
-                    if (args.Data != null)
+                    if (args.Data != null) {
```

Comment:
Can you make the opening braces start on new line please (here and in other places)?